src/main/java/com/cloudant/client/api/Database.java

```diff
@@ -245,6 +246,9 @@ public void createIndex(String indexDefinition) {
 				list.add(t);
 			}
 			return list;
+		 } catch (UnsupportedEncodingException e) {
+			// This should never happen as every implementation of the java platform is required to support UTF-8.
+			return null;

```

Comment:
Okay, so what I think I'd do then, is keep the comments about the exception should never be thrown and just have `throw new RuntimeException(e)` instead of `return null`